### DISCUSSION TITLE
Input time with 15 minutes step

### DISCUSSION THREAD

**Discussion by lucapu88** [bootstrap-vue/bootstrap-vue#6218] [Q&A] [answered]
I have this input time component and I would like the minutes to be marked in steps of 15... currently as I wrote it doesn't work... is there a solution?

```
<b-form-input
     minutes-step="15"
    type="time" >
</b-form-input>
```

**Comment by Hiws** [comment, marked as answer]
You can specify how much it should step using the `step` attribute.
You need to provide the step in seconds. So 15 minutes would be 900 seconds.

Reference: https://html.spec.whatwg.org/multipage/input.html#time-state-(type=time)

**Reply by lucapu88** [reply]
_For anyone who will read this thread:_ **the problem has not been solved!**
it works if you type with the keyboard, but if you click on the timePicker, the minutes appear one by one and not in steps of 15 ... so this IS NOT A SOLUTION.

**Reply by Hiws** [reply]
`<b-form-input>` renders a native `<input>` element, which means we don't have control over how it works.
This is all left up to the browser.

So it's up to the browser to decide how it handles the `step` attribute.
However, the `step` attribute is the closest you'll get with a native `<input>`.
